tests/backends/postgresql/commands/test_init.py

```diff
@@ -32,3 +32,30 @@ def test_prepare(rc: RawConfig):
     assert [type(c).__name__ for c in table.constraints] == [
         'PrimaryKeyConstraint',
     ]
+
+
+def test_prepare_model_ref_unique_constraint(rc: RawConfig):
+    context, manifest = load_manifest_and_context(rc, '''
+    d | r | b | m | property   | type    | ref       | level | access
+    example                    |         |           |       |
+      |   |   | Continent      |         | id        | 4     |
+      |   |   |   | id         | integer |           | 3     | open
+      |   |   |   | name       | string  |           | 3     | open
+      |   |   | Country        |         | id, name  | 4     |
+      |   |   |   | id         | integer |           | 3     | open
+      |   |   |   | name       | string  |           | 3     | open
+    ''')
+    model_single_unique = manifest.models['example/Continent']
+    backend = model_single_unique.backend
+    commands.prepare(context, backend, model_single_unique)
+    table = backend.get_table(model_single_unique)
+    assert any(
+        [table.c['id']] == list(constraint.columns) for constraint in table.constraints if
+        type(constraint).__name__ == 'UniqueConstraint')
+
+    model_multiple_unique = manifest.models['example/Country']
+    commands.prepare(context, backend, model_multiple_unique)
+    table = backend.get_table(model_multiple_unique)
+    assert any(
+        [table.c['id'], table.c['name']] == list(constraint.columns) for constraint in table.constraints if
+        type(constraint).__name__ == 'UniqueConstraint')
```

Comment:
Please, add tests for handling unique constraint errors, when trying to write a duplicate value on a column or columns, that have unique constraints.